### PR TITLE
Fixes #1647 - split font short form for webkit

### LINKED ISSUE #1647: Some icons have disappeared
Some icons have disappeared: for example, Home, log in (header)

### PR DESCRIPTION
**Pull request by zoepage**
r? @miketaylr @magsout

### CODE REVIEW

File: webcompat/static/css/development/components/icon-font.css [hunk 1]
```diff
@@ -13,8 +13,16 @@
 .wc-Icon {
   display: inline-block;
   vertical-align: middle;
-  font: normal normal 400 normal 1em/1 webcompat;
+
+  /* to prevent a webcompat issue, we do not use the short for font form here right now
+  webkit has a css2 implementation in 10.1.1. regarding to that spec */
```
Comment (miketaylr) [inline webcompat/static/css/development/components/icon-font.css]: What is the actual issue? `webkit has a css2 implementation` is a confusing comment -- is it a bug in a particular part of css3 shorthand value parsing? Or something else?

Comment (miketaylr) [inline webcompat/static/css/development/components/icon-font.css]: (is this the bug? https://bugs.webkit.org/show_bug.cgi?id=165594)

**Comment by zoepage** [comment]
Yes, it is that one

File: webcompat/static/css/development/components/icon-font.css (hunk 1 above)
Comment (miketaylr) [inline webcompat/static/css/development/components/icon-font.css]: @zoepage wrote:
> Yes, it is that one

OK, can we update the comment then. It's not accurate to say Safari implements CSS2 for `font`  -- it's just a regression that we need to work around. Thanks!

**Comment by zoepage** [comment]
Will do. Thanks

**Comment by miketaylr** [comment]
👍

**Pull request merged**